### DISCUSSION TITLE
네이버인증서/핸드폰인증 방법 무관하게 잘 동작하나요?

### DISCUSSION THREAD

**Discussion by ghost** [changdoc/naver-vaccine-macro#73] [Q&A] [answered]
인증서를 이용하면 이렇게 개인정보 수집 및 제공 동의가 필요없고
![1](https://user-images.githubusercontent.com/38450291/126611698-238b9ae7-5c11-4269-b0c0-2bd93e1b7e1a.JPG)

핸드폰인증을 하면 이렇게 개인정보 수집 및 제공 동의가 필요한데
![2](https://user-images.githubusercontent.com/38450291/126611704-6f9efbc4-63d0-4482-a340-f3792c534645.JPG)

혹시 두 방법 모두 가능한 것인가요? 네이버 인증서 방식은 1인당 1계정만 가능한데,
핸드폰 인증방식은 1인당 3계정까지 지원하다보니
현재 블락 방식이 계정 차단인 것 같은데, 이렇게 하면 2개를 동시에 켜놓을 수도 있을 것 같아서요
(특히 비슷한 시간대에 예약을 하는 병원들이 있다보니 더더욱 필요하네요)

**Comment by changdoc** [comment, marked as answer]
네 인증서를 사용해도 정보 제공 동의 철회를 하면 아래 화면 처럼 나오는데 예약 신청 정상적으로 시도하는 부분 확인했습니다.

**Reply by murokim** [reply]
네이버 인증서 안하고 휴대폰 인증만 해도 정상작동 하나요?
